### ISSUE TITLE
Failed to load icinga_zone "master"

### ISSUE THREAD

**Issue by PAStheLoD** [Icinga/icingaweb2-module-director#883]
Hello,

I know there's an old bug with the same name ( https://github.com/Icinga/icingaweb2-module-director/issues/224 ), but I'm using the current master (6234648a1ffe74a7239b2ab30a3bb3b7afafc6dc) of director and master of icingaweb2 (f7a8cf8a6ee8d94ba2997348907fbbc9521aa687).

```
root@mon:/opt/icingaweb2# bin/icingacli director kickstart run
ERROR: Icinga\Exception\NotFoundError in /opt/icingaweb2/modules/director/library/Director/Data/Db/DbObject.php:589 with message: Failed to load icinga_zone "master"
```

And the same thing happens using the web UI. And there are no pending migrations.

Thank you in advance!

**Comment by acesuares** [comment]
Is this the same error?

    Failed to load icinga_zone "MY HOST" (DbObject.php:589)

**Comment by cyberkov** [comment]
Hello!

This problem applies to me as well and I ran a git bisect against it:
```
root@viesv026:/usr/share/icingaweb2/modules/director# git bisect start HEAD v1.3.0
Bisecting: 36 revisions left to test after this (roughly 5 steps)
[58f7b16f21fa1a29a37fd9e48567d04f4f870ab6] doc/changelog: mention multi-edit
root@viesv026:/usr/share/icingaweb2/modules/director# git bisect run ../test.sh
running ../test.sh
+ mysql -e 'drop database if exists icingadirector; create database icingadirector'
+ icingacli director migration run
+ icingacli director kickstart run
Bisecting: 18 revisions left to test after this (roughly 4 steps)
[158ce18302c8719e7c65be0228a073e2a250457f] SyncUtilsTest: use nicer test name
running ../test.sh
+ mysql -e 'drop database if exists icingadirector; create database icingadirector'
+ icingacli director migration run
+ icingacli director kickstart run
Bisecting: 9 revisions left to test after this (roughly 3 steps)
[58f77d39a910c231cb992543c1eb0124d409d184] doc/agents: update 24-Working-with-agents.md
running ../test.sh
+ mysql -e 'drop database if exists icingadirector; create database icingadirector'
+ icingacli director migration run
+ icingacli director kickstart run
ERROR: Icinga\Exception\NotFoundError in /usr/share/icingaweb2/modules/director/library/Director/Data/Db/DbObject.php:589 with message: Failed to load icinga_zone "master"
Bisecting: 4 revisions left to test after this (roughly 2 steps)
[242bff401134d482d9d184d61abf0b76c734d84b] KickstartHelper: handle renamed master
running ../test.sh
+ mysql -e 'drop database if exists icingadirector; create database icingadirector'
+ icingacli director migration run
+ icingacli director kickstart run
ERROR: Icinga\Exception\NotFoundError in /usr/share/icingaweb2/modules/director/library/Director/Data/Db/DbObject.php:589 with message: Failed to load icinga_zone "master"
Bisecting: 1 revision left to test after this (roughly 1 step)
[cb9fff90ef4d778feed9d51f571bb662ddeeb974] IcingaObject: add new static helper, expose...
running ../test.sh
+ mysql -e 'drop database if exists icingadirector; create database icingadirector'
+ icingacli director migration run
+ icingacli director kickstart run
Bisecting: 0 revisions left to test after this (roughly 0 steps)
[489d4ab1a380daf69787fe97a97bf72966b6da0f] KickstartForm: let parent handle exceptions
running ../test.sh
+ mysql -e 'drop database if exists icingadirector; create database icingadirector'
+ icingacli director migration run
+ icingacli director kickstart run
242bff401134d482d9d184d61abf0b76c734d84b is the first bad commit
commit 242bff401134d482d9d184d61abf0b76c734d84b
Author: Thomas Gelf <thomas@gelf.net>
Date:   Mon Mar 13 22:06:10 2017 +0100

    KickstartHelper: handle renamed master

    * fix zone cleanup
    * delete obsolete endpoints
    * re-structure code
    * remove obsolete Db helper

    fixes #842

:040000 040000 12a74211a2ae0adb49d2ecd28039ec20d573019d 3f36556f4f2519a0049613ff697f286299699e0f M      library
bisect run success
```
Unfortunately I do not understand the code enough to actually provide a workaround for the problem introduced in 242bff401134d482d9d184d61abf0b76c734d84b
If I can be of any help please let me know @Thomas-Gelf :)

Cheers Hannes

**Comment by acesuares** [comment]
https://acesuares.github.io/install-icinga2-director-on-virtualbox-with-vagrant/ now with 28 easy steps!

**Comment by widhalmt** [comment]
I had an error during rerunning Kickstart wizard like "Failed to load" as well. Thought it was about having `_` in the zones name but this seems not to be the problem. I could easily reload the zone and rerun the Kickstart wizard again - now it's working.

**Comment by Thomas-Gelf** [comment]
This should have been fixed with d41fbbf - in case it still appears please let me know.
